pkg/util/webhooks/validating-webhooks/validating-webhook.go

```diff
@@ -19,17 +20,10 @@ type admitter interface {
 	Admit(context.Context, *admissionv1.AdmissionReview) *admissionv1.AdmissionResponse
 }
 
-type AlwaysPassAdmitter struct {
-}
-
```

Comment:
Could you please add reasoning for the removal of this struct?

Reply:
it was not in use